packages/engine/bin/cli/src/main.rs

```diff
@@ -11,73 +11,77 @@ pub mod exsrv;
 pub mod manifest;
 pub mod process;
 
+use std::path::PathBuf;
+
 use anyhow::Result;
+use clap::{AppSettings, Args, Parser, Subcommand};
 use experiment::run_experiment;
-use structopt::StructOpt;
 
 use crate::exsrv::create_server;
 
-/// The hEngine Command line interface.
-///
-/// Can run single or simple experiments.
-#[derive(Debug, StructOpt)]
-pub struct Args {
+/// Arguments passed to the CLI
+#[derive(Debug, Parser)]
+#[clap(about, version, author)]
+#[clap(global_setting(AppSettings::PropagateVersion))]
+#[clap(setting(AppSettings::UseLongFormatForHelpSubcommand))]
+pub struct App {
```

Comment:
Is calling it App convention? Args seemed to make more sense to me at least

Reply:
`clap` adds a trait called `Args` which is derived on some of the structs. I just used `App` as it's used in `clap` documentation as well, we could also just import use `clap::Args` directly, no strong opinion.

Reply:
I think lets go with `clap::Args` for the trait, because `App` doesn't make a tonne of sense to me and it's inconsistent with the other file anyway where we still have `Args`

Reply:
Fair enough!